pages/product.html

```diff
@@ -0,0 +1,410 @@
+        class="product__gradient"
+      />
+
+      <div class="product-cards">
+        <div class="product-cards__card">
+          <img
+            class="product-cards__image"
+            src="../assets/images/icon1.png"
+            alt=""
+          />
+          <h3 class="product-cards__title">
+            High security to protect from piracy
+          </h3>
+          <p class="product-cards__description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+
+        <div class="product-cards__card">
+          <img
+            class="product-cards__image"
+            src="../assets/vectors/Icon-home-2.svg"
+            alt=""
+          />
+          <h3 class="product-cards__title">
+            High security to protect from piracy
+          </h3>
+          <p class="product-cards__description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+
+        <div class="product-cards__card">
+          <img
+            class="product-cards__image"
+            src="../assets/vectors/Icon-home-3.svg"
+            alt=""
+          />
+          <h3 class="product-cards__title">
+            High security to protect from piracy
+          </h3>
+          <p class="product-cards__description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+
+        <div class="product-cards__card">
+          <img
+            class="product-cards__image"
+            src="../assets/vectors/Icon-home-4.svg"
+            alt=""
+          />
+          <h3 class="product-cards__title">
+            High security to protect from piracy
+          </h3>
+          <p class="product-cards__description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+
+        <div class="product-cards__card">
+          <img
+            class="product-cards__image"
+            src="../assets/vectors/Icon-home-5.svg"
+            alt=""
+          />
+          <h3 class="product-cards__title">
+            High security to protect from piracy
+          </h3>
+          <p class="product-cards__description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+
+        <div class="product-cards__card">
+          <img
+            class="product-cards__image"
+            src="../assets/vectors/Icon-home-6.svg"
+            alt=""
+          />
+          <h3 class="product-cards__title">
+            High security to protect from piracy
+          </h3>
+          <p class="product-cards__description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy.
+          </p>
+        </div>
+      </div>
```

Comment:
## prohibition of nested block
`products-cards__X` change even the element too.